relay-server/src/actors/project.rs

```diff
@@ -20,16 +19,15 @@ use relay_system::BroadcastChannel;
 
 use crate::actors::envelopes::{EnvelopeManager, SendMetrics};
 use crate::actors::outcome::{DiscardReason, Outcome};
-use crate::actors::processor::{EnvelopeProcessor, ProcessEnvelope};
-use crate::actors::project_cache::{
-    AddSamplingState, CheckedEnvelope, ProjectCache, RequestUpdate,
-};
+#[cfg(feature = "processing")]
+use crate::actors::processor::EnvelopeProcessor;
```

Comment:
We have the following in the code: https://github.com/getsentry/relay/blob/381c8936e168895fd241990719a452133daaa993/relay-server/src/actors/project.rs#L798-L807
which already guards some of the code behind the feature flag, so the import must be there as well, otherwise the compiler will complain.